docs/flashbots-auction/searchers/advanced/rpc-endpoint.mdx

```diff
@@ -7,7 +7,7 @@ import TabItem from "@theme/TabItem";
 
 ## How to interact directly with the Flashbots RPC endpoint
 
-For advanced users, you can interact with the relay's RPC endpoint at `relay.flashbots.net`. The relay provides new JSON-RPC methods for interfacing with Flashbots which are documented below:
+For advanced users, you can interact with the relay's RPC endpoint at `protect.flashbots.net`. The Protect API provides JSON-RPC methods for interfacing with Flashbots which are documented below:
```

Comment:
name check

Reply:
removed "Protect" here -- this API ("bundle API" or "bundle relay" or whatever) isn't currently designated under the Protect brand